courses/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+# Create your models here.
+
+class Languages(models.Model):
+    language_name = models.CharField(max_length=20)
```

Comment:
add slugs field which shall be autogenerated/autoupdated on save overrides. Refer django documentation to find django methods to create a slug.